test/unit/utils/telemetry/collector_test.rb

```diff
@@ -0,0 +1,46 @@
+require 'inspec/utils/telemetry'
+require_relative '../../../helper.rb'
+
+class TestTelemetryCollector < Minitest::Test
+  def setup
+    @collector = Inspec::Telemetry::Collector.instance
+    @collector.reset
+  end
+
+  def test_collector_singleton
+    assert_equal Inspec::Telemetry::Collector.instance, @collector
+  end
+
+  def test_add_data_series
+    data_series = Inspec::Telemetry::DataSeries.new('/resource/File')
+    assert @collector.add_data_series(data_series)
```

Comment:
What is being asserted here? `assert` checks to see if a value is truthy; can you say anything stronger about the return value of `add_data_series`?

Reply:
I'll add another test to check that the data series was added correctly.